integration-tests/soak/tests/ocr_test.go

```diff
@@ -0,0 +1,225 @@
+package soak_test
+
+// revive:disable:dot-imports
+import (
+	"context"
+	"flag"
+	"fmt"
+	"math/big"
+	"math/rand"
+	"os"
+	"strconv"
+	"time"
+
+	caigotypes "github.com/dontpanicdao/caigo/types"
+
+	"github.com/dontpanicdao/caigo/gateway"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/rs/zerolog/log"
+	"github.com/smartcontractkit/chainlink/integration-tests/actions"
+	client "github.com/smartcontractkit/chainlink/integration-tests/client"
+
+	"github.com/smartcontractkit/chainlink-starknet/integration-tests/common"
+	"github.com/smartcontractkit/chainlink-starknet/ops/gauntlet"
+	"github.com/smartcontractkit/chainlink-starknet/relayer/pkg/chainlink/ocr2"
+)
+
+var (
+	keepAlive bool
+)
+
+func init() {
+	flag.BoolVar(&keepAlive, "keep-alive", false, "enable to keep the cluster alive")
+}
+
+var _ = Describe("StarkNET OCR suite @ocr", func() {
+	var (
+		err                 error
+		t                   *common.Test
+		serviceKeyL1        = "Hardhat"
+		serviceKeyL2        = "starknet-dev"
+		serviceKeyChainlink = "chainlink"
+		chainName           = "starknet"
+		chainId             = gateway.GOERLI_ID
+		cfg                 *common.Common
+		decimals            = 9
+		mockServerVal       = 900000000
+		roundWaitTimeout    = time.Hour * 72
+		nodeCount           = 5
+	)
+
+	BeforeEach(func() {
+		By("Gauntlet preparation", func() {
+
+			// Checking if count of OCR nodes is defined in ENV
+			nodeCountSet, nodeCountDefined := os.LookupEnv("NODE_COUNT")
+			if nodeCountDefined == true {
+				nodeCount, err = strconv.Atoi(nodeCountSet)
+				if err != nil {
+					panic(fmt.Sprintf("Please define a proper node count for the test: %v", err))
+				}
+			}
+
+			// Checking if TTL env var is set to adjust duration to custom value
+			ttlValue, ttlDefined := os.LookupEnv("TTL")
```

Comment:
Keep in mind that not all valid `Go` duration strings are valid for `janitor` we are using, please use `ShortDur` from `chainlink-env`